app/views/payload/edit.html

```diff
@@ -36,22 +36,51 @@ <h3 class="m-b-xs">Payload</h3>
                             <div class="form-group">
                                 <label class="control-label">Script src</label><label class="shortboost" domain="{%data domain}" class="control-label"></label>
                                 <input class="form-control scriptsrc" disabled
-                                    value='&lt;script src=//{%data domain}&gt;&lt;/script&gt;'><br>
+                                    value="&lt;script src=//{%data domain}&gt;&lt;/script&gt;"><br>
 
                                 <label class="control-label">Script src with closing tag</label>
                                 <input class="form-control scriptsrc" disabled
-                                    value='"&gt;&lt;script src=//{%data domain}&gt;&lt;/script&gt;'><br>
+                                    value="&quot;&gt;&lt;script src=//xss.nullvoid.me&gt;&lt;/script&gt;"><br>
 
```

Comment:
This should be updates to use `{%data domain}`

Reply:
Updated, please squash on merge to upstream.